### ISSUE TITLE
Brave STILL crashes 5 times a day.

### ISSUE THREAD

**Issue by RanaInteractive** [brave/brave-browser#40487]
Brave STILL crashes 5 times a day... with only one window open, this one:

(https://community.brave.com/t/brave-keeps-crashing-and-is-getting-worse/557686)

Yet I have never heard a single thing form anyone at Brave. Good job, Well done Brave?

**Comment by rebron** [comment]
likely dupe of https://github.com/brave/brave-browser/issues/39101

**Comment by bondeabhijeet** [comment]
@RanaInteractive kindly consider following these [steps](https://github.com/brave/brave-browser/issues/40532#issuecomment-2297102776)

**Comment by bsclifton** [comment]
@RanaInteractive do you have more information about your setup? Are you on macOS? What version? 

https://github.com/brave/brave-browser/issues/39101 does have an interesting crash we can use help narrowing down. Seems to be a serverTrust call being made which crashes in boringssl. Are you a developer? ex: do you have homebrew installed / possibly installed some openssl libraries or dependencies?

Please let us know more information - we don't have enough information to troubleshoot the situation unfortunately.

**Comment by RanaInteractive** [comment]
All of the relevant info is here… starting on July 11.

https://community.brave.com/t/brave-keeps-crashing-and-is-getting-worse/557686 <https://community.brave.com/t/brave-keeps-crashing-and-is-getting-worse/557686>

I’ll answer some of your other questions here:

No… I am not a developer, just a user of your browser

No Homebrew installed

And this I don’t know anything about… possibly installed some openssl libraries or dependencies?

Thank you for your attention… John

**Comment by bsclifton** [comment]
@RanaInteractive I replied there - but maybe you can help share here too. Are there any sites in particular that (when visited) seem to cause this? That is something I would look out for please. We need help to reproduce the problem so that we can trap the error condition. 

We only know it's happening for serverTrust validation - which is vague. We can't disable this for folks for obvious reasons. If we are able to find a handful of sites that it always happens on, we might be able to find what part of serverTrust validation is failing and see if there is a possible work-around.   Right now, we don't have enough to do anything

**Comment by bsclifton** [comment]
Because this is the same crash as https://github.com/brave/brave-browser/issues/39101, I'm going to close this issue as a duplicate.

Please subscribe to https://github.com/brave/brave-browser/issues/39101 for updates. I'll reply here too, of course. So you're welcome to share that extra info and I can drop it in the other ticket.

**Comment by RanaInteractive** [comment]
I’m really not sure what tickets are open… this is all VERY confusing.

Sometimes it crashes when you first open it, as it is starting up, sometime it crashes while only the screen (monitor) is sleeping, or while the actual computer was asleep (say overnight), sometimes it seems to crash more, or the minute I open more than one window and move one of them to a different window using Mission Control in macOS, for a while it seemed to crash more often when I had multiple windows open, and was using using Mission Control, so stoped using Mission Control, but this did not stop the crashes all together, sometimes it stays open all day, sometimes it crashes five times in 30 minutes. I can have just one window open, for instance with this address, https://community.brave.com/t/brave-keeps-crashing-and-is-getting-worse/557686, and it will crash. 

And there are no specific sites, or domains that cause it to crash. It seems to be totally random.

**Comment by RanaInteractive** [comment]
Is it possible that this could be caused by extensions?

When I went to my extensions there were additional ad Blocking extensions installed (and I couldn’t remember why I had installed them, since this is supposed to be built-in to Brave in the first place). So I deleted everything and the browser seemed to be doing better, at least during the hour or so I was testing it. But now of course Brave is no longer blocking ads… which is the only reason I started using Brave in the first place.

The latest version of Brave I’m “allowed" to use is version 1.57.64, and I’m sure that is probably what I have installed simply because I would have looked that up…

But... I can’t tell you for sure because every time I open Brave and and try to get the version number by going to the Brave > About Brave menu… the browser immediately crashes immediately!

I can open any of the other links in the same drop down (Preferences for instance), but "About Brave" immediately crashes the browser.

- - -

The latest version of the Brave browser that will run on macOS Catalina (Version 10.15.7) is Brave version 1.57.64. This version is the last supported for macOS 10.15, as subsequent updates (starting from version 1.58) require macOS 11 (Big Sur) or later to function properly.

If you're currently using Brave on Catalina, it's advisable to stick with version 1.57.64 to avoid compatibility issues, as newer versions will not be available for your operating system.
